spec/gate/indexSpec.js

```diff
@@ -89,4 +89,27 @@ describe('UserGate', function() {
       expect(gate.allows('bar@mixmax.com')).toBe(true);
     });
   });
+
+  describe('uniform sample', () => {
```

Comment:
Can we add some test(s) of this function's uniformity? Perhaps port [this `TrialUtils` test](https://github.com/mixmaxhq/emailer-chrome-extension-source/blob/fd32ee26cf1a8d0d7a5642a3ef4b889fbdf7f9d8/test/spec/utils/TrialUtilsSpec.js#L57).

Reply:
https://github.com/mixmaxhq/user-gate/pull/5/commits/bbaee3d607cd7dc45dd72ebe13de72fb55c7c6d1